File: msgraph-mail/dotnet/Users/Importance.cs
using Kiota.Abstractions.Serialization;
using System;
using System.Collections.Generic;
using System.Linq;
namespace Graphdotnetv4.Users {
    public class Importance : IParsable<Importance> {
        public IDictionary<string, Action<Importance, IParseNode>> DeserializeFields => new Dictionary<string, Action<Importance, IParseNode>> {
        };
        public void Serialize(ISerializationWriter writer) {
        }
    }
}

File: msgraph-mail/typescript/src/users/importance.ts
import {SerializationWriter, ParseNode, Parsable} from '@microsoft/kiota-abstractions';

export class Importance implements Parsable<Importance> {
    public serialize (writer: SerializationWriter) : void {
    };
    public deserializeFields () : Map<string, (item: Importance, node: ParseNode) => void> {
        return new Map<string, (item: Importance, node: ParseNode) => void>([
        ]);
    };
}
